portfolio/src/main/webapp/script.js

```diff
@@ -16,3 +16,16 @@ window.addEventListener('DOMContentLoaded', (event) => {
       }
   }
 });
+
+function getComments() {
+    fetch('/data').then(response => response.json()).then((comments) => {
+        
+        const commentElement = document.getElementById('comment-container');
+        commentElement.innerHTML = '';
+
+        for(var i = 0; i < comments.length; i++) {
+            commentElement.append(comments[i] + " ");           
+        }
+
+    });
```

Comment:
Nit: I think your spacing for this function is different from the rest of the file?